README.md

```diff
@@ -1,239 +1,156 @@
-# Auth0 Java MVC Commons
+![Auth0 SDK to add authentication to your Java Servlet applications.](https://cdn.auth0.com/website/sdks/banners/auth0-java-mvc-common-banner.png)
 
 [![CircleCI](https://img.shields.io/circleci/project/github/auth0/auth0-java-mvc-common.svg?style=flat-square)](https://circleci.com/gh/auth0/auth0-java-mvc-common/tree/master)
 [![Coverage Status](https://img.shields.io/codecov/c/github/auth0/auth0-java-mvc-common.svg?style=flat-square)](https://codecov.io/github/auth0/auth0-java-mvc-common)
-[![License](http://img.shields.io/:license-mit-blue.svg?style=flat)](http://doge.mit-license.org)
-[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fauth0%2Fauth0-java-mvc-common.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fauth0%2Fauth0-java-mvc-common?ref=badge_shield)
+[![License](http://img.shields.io/:license-mit-blue.svg?style=flat)](https://doge.mit-license.org/)
+[![Maven Central](https://img.shields.io/maven-central/v/com.auth0/mvc-auth-commons.svg?style=flat-square)](https://mvnrepository.com/artifact/com.auth0/mvc-auth-commons)
+[![javadoc](https://javadoc.io/badge2/com.auth0/auth0-java-mvc-common/javadoc.svg)](https://javadoc.io/doc/com.auth0/mvc-auth-commons)
 
-A Java Jar library that makes easier to integrate Auth0 Authentication on MVC applications.
+:books: [Documentation](#documentation) - :rocket: [Getting Started](#getting-started) - :computer: [API Reference](#api-reference) :speech_balloon: [Feedback](#feedback)
 
-See the [Java Servlet Quickstart](https://auth0.com/docs/quickstart/webapp/java) to learn how to use this library in a Servlet application.
+## Documentation
 
-> If you are using Spring Boot 2, it is recommended to use the OIDC support available in Spring, instead of using this library. See the [Spring Boot Quickstart](https://auth0.com/docs/quickstart/webapp/java-spring-boot) for more information.
+- [Quickstart](https://auth0.com/docs/quickstart/webapp/java) - our interactive guide for quickly adding login, logout and user information to a Java Servlet application using Auth0.
+- [Sample App](https://github.com/auth0-samples/auth0-servlet-sample/tree/master/01-Login) - a sample Java Servlet application integrated with Auth0.
+- [Examples](./EXAMPLES.md) - code samples for common scenarios.
+- [Docs site](https://www.auth0.com/docs) - explore our docs site and learn more about Auth0.
 
-## Download
+## Getting Started
 
-Via Maven:
+### Requirements
+
+Java 8 or above and `javax.servlet` version 3.
+
+> If you are using Spring, we recommend leveraging Spring's OIDC and OAuth2 support, as demonstrated by the [Spring Boot Quickstart](https://auth0.com/docs/quickstart/webapp/java-spring-boot).
+
+### Installation
+
+Add the dependency via Maven:
 
 ```xml
 <dependency>
-    <groupId>com.auth0</groupId>
-    <artifactId>mvc-auth-commons</artifactId>
-    <version>1.9.2</version>
+  <groupId>com.auth0</groupId>
+  <artifactId>auth0-java-mvc-common</artifactId>
```

Comment:
Just to be sure: Is the change to `artifactId` expected?

Reply:
oh, good catch! No, that is not expected. New commit pushed to fix.